FluentUI.Demo/src/main/java/com/microsoft/fluentuidemo/demos/TooltipActivity.kt

```diff
@@ -93,7 +93,7 @@ class TooltipActivity : DemoActivity(), Tooltip.OnDismissListener {
         }
 
         tooltip_anchor_custom_view.setOnClickListener {
-            tooltip = Tooltip(this).show(
+            tooltip = Tooltip(this).setFocusable(true).show(
```

Comment:
Why setFocusable set to true explicitly here ?. Should not it get set explicitly by user.